### PR TITLE
DATAUP-234 remove old integration test cruft

### PR DESCRIPTION
**Pull request by briehl**
# Description of PR purpose/changes

DO NOT MERGE before merging #1863 , as this removes the CI testing stuff, and expects the github actions to be present.

This removes code and references to older selenium code that's no longer used. It also removes some references to PhantomJS, which is also no longer used.

This is semi-dependent on #1863 and should be rebased if that's accepted first.

# Jira Ticket / Issue #
e.g. https://kbase-jira.atlassian.net/browse/DATAUP-X
- [x] Added the Jira Ticket to the title of the PR e.g. (DATAUP-69 Adds a PR template)

# Testing Instructions
* Details for how to test the PR: 

1. Tests shouldn't run in Travis
2. Tests should run and pass in Github actions.

- [ ] Tests pass in github actions and locally

### CODE REVIEW

**Review by eamahanna — APPROVED** [review]
All of these removals make sense to me. Thanks for doing this!

**Comment by briehl** [comment]
Great. I'll hold off on merging until the github actions PR gets merged.

**Pull request merged**
